lua/nvim-lsp-installer/log.lua

```diff
@@ -1,18 +1,175 @@
-local M = {}
```

Comment:
Do we need to attribute this code to the original author(s) here? It looks like it's under the MIT license

Reply:
It's totally fine to remove it 👍